src/app/not-found.tsx

```diff
@@ -1,11 +1,27 @@
-import Link from "next/link";
-
 export default function NotFound() {
   return (
     <div>
-      <h2>Not Found</h2>
-      <p>Could not find requested resource</p>
-      <Link href="/">Return Home</Link>
+      <div className="flex flex-col items-center my-10">
+        <h1 className="text-[2.5rem] font-pressstart uppercase leading-tight">
+          <span>404</span>
+        </h1>
+        <h2 className="uppercase md:text-2xl font-normal">
+          <span>Page not found</span>
+        </h2>
+        <hr className="w-1/4 max-w-[160px] border-breadpink-shaded my-6" />
+        <div className="flex flex-col items-center max-w-[250px]">
+          <p className="leading-tight text-center">
+            Post-Capitalism can only be achieved through perserverance.
+          </p>
+          <p className="leading-tight text-center">Please try again later.</p>
+        </div>
+        <a
+          href="/"
+          className="px-5 py-2 my-6 text-base rounded-xl inline-block font-bold transition-all bg-breadviolet-shaded dark:bg-breadpink-shaded text-breadgray-ultra-white dark:text-breadgray-og-dark disabled:bg-opacity-60"
```

Comment:
I combined all the relevant button classes to achieve this with an`<a>` tag since its just a link to home page -- but not sure if there's a better way to have done this

Reply:
Looks good although there is a reusable `<Button>` component in `core/components` and I think the styles are the same.

I'd probably suggest sharing the styles and exporting a `<ButtonLink>` component from the same file so it's in the same place if styles change.

Reply:
yup, sounds right! 

I made a new ButtonLink component which just imports things already defined in Button, but renders an `a` tag. 

I'm blocked from merging until you post an approval on a review - can you take a quick look at my latest changes and if approve if all looks good?

Reply:
@subject026 whoops, remembered the process is just to merge to dev. will do that now!